docs/docs/build/deploy/foundry.md

```diff
@@ -0,0 +1,152 @@
+
+```solidity
+
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.4;
+
+contract Coin {
+    // The keyword "public" makes variables
+    // accessible from other contracts
+    address public minter;
+    mapping(address => uint) public balances;
+
+    // Events allow clients to react to specific
+    // contract changes you declare
+    event Sent(address from, address to, uint amount);
+
+    // Constructor code is only run when the contract
+    // is created
+    constructor() {
+        minter = msg.sender;
+    }
+
+    // Sends an amount of newly created coins to an address
+    // Can only be called by the contract creator
+    function mint(address receiver, uint amount) public {
+        require(msg.sender == minter);
+        balances[receiver] += amount;
+    }
+
+    // Errors allow you to provide information about
+    // why an operation failed. They are returned
+    // to the caller of the function.
+    error InsufficientBalance(uint requested, uint available);
+
+    // Sends an amount of existing coins
+    // from any caller to an address
+    function send(address receiver, uint amount) public {
+        if (amount > balances[msg.sender])
+            revert InsufficientBalance({
+                requested: amount,
+                available: balances[msg.sender]
+            });
+
+        balances[msg.sender] -= amount;
+        balances[receiver] += amount;
+        emit Sent(msg.sender, receiver, amount);
+    }
+}
+```
+
+## Compiling Contract 
+
+- To compile contract run the followling command. 
+
+```shell
+$ forge build
+```
+
+## Deploying Contract
+
+- To deploy the contract via the terminal, you'll need your private key. If you're using MetaMask, be cautious when exporting your private key as it can be risky. Checkout [article](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to get your private key from metamask.
+
+- Deploy compiled smart contract 
+
+```shell
+$ forge create --rpc-url <enter_bob_rpc_url> --private-key <enter_private_key> src/<enter_contract_name>.sol:<enter_contract_name>
+```
+
+- For coin smart contract 
+
+```shell
+$ forge create --rpc-url wss://l2-fluffy-bob-7mjgi9pmtg.t.conduit.xyz --private-key <enter_private_key> src/Coin.sol:Coin
+```
+
+- The rpc url can be changed checkout [conduit](https://app.conduit.xyz/published/view/fluffy-bob-7mjgi9pmtg) to get the latest endpoints. 
+
+- The output in the terminal should look similar to this
+
+```shell
+Deployer: 0xd8a0bb324b46D89C105BA98C402dF0972b9164Af
+Deployed to: 0xbd56c1FFF2d2073F84825D582808885dbB2085C6
+Transaction hash: 0x263ead5ea07e6122d4d1fe6544158502d278b23e86b2a5b143770b82eead1588
+```
+
+## Interaction with Contract
+
+- Checkout [testnet explorer](https://explorerl2-fluffy-bob-7mjgi9pmtg.t.conduit.xyz) to get contract details using Transaction hash. 
+
+![Contract details on Explorer Image](../../../static/img/deployed_contract_on_foundry.png)
```

Comment:
Instead of putting images in the static folder, they should be directly in the same folder as the docs. See, e.g., https://github.com/bob-collective/bob/tree/master/docs/docs/learn/introduction